runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeConfig.java

```diff
@@ -56,15 +56,4 @@ default boolean enableNotifications()
 	{
 		return true;
 	}
-
-	@ConfigItem(
-		position = 3,
-		keyName = "notificationDelay",
-		name = "Notification Delay",
-		description = "Number of seconds between notifications on offer updates"
-	)
-	default int notificationDelay()
-	{
-		return 5;
```

Comment:
Does the game have something built in to prevent this?

Reply:
I dont think we need anything to prevent instant notification as the message do not appears if you have GE interface still opened (just tested it to be sure).

Reply:
The game doesn't send multiple notifications within a small set of time, from what I've seen. So you shouldn't receive a BUYING/BOUGHT notification right after each other.